www-plugins/passff-host/passff-host-1.0.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_4 )
+
+inherit python-single-r1
+
+DESCRIPTION="Host app for the PassFF WebExtension"
+HOMEPAGE="https://github.com/passff/passff-host"
+SRC_URI="
+	https://github.com/passff/passff-host/releases/download/${PV}/passff.py   -> ${P}.py
+	https://github.com/passff/passff-host/releases/download/${PV}/passff.json -> ${P}.json
+"
+
+LICENSE="GPL-2"
```

Comment:
FYI, I've filed https://github.com/passff/passff-host/issues/16 since the project seems to lack clear notice. However, until upstream updates that this is good.